src/absil/ilreflect.fs

```diff
@@ -1235,9 +1271,12 @@ let emitCode cenv modB emEnv (ilG:ILGenerator) code =
 let emitLocal cenv emEnv (ilG : ILGenerator) (local: ILLocal) =
     let ty = convType cenv emEnv  local.Type
     let locBuilder = ilG.DeclareLocalAndLog(ty, local.IsPinned)
+#if NO_PDB_WRITER
```

Comment:
Can you clarify the difference between `NO_PDB_WRITER` and `RESHAPED_REFEMIT`? Here, the "add some debug info via ref emit" stuff is excluded using the former, but above it's excluded using the latter.